examples/seismic/acoustic/constant_example.py

```diff
@@ -24,66 +27,76 @@ def source(t, f0):
     return (1-2.*r**2)*np.exp(-r**2)
 
 
-def run(dimensions=(50, 50, 50), spacing=(20.0, 20.0, 20.0), tn=1000.0,
-        time_order=2, space_order=4, nbpml=40, dse='advanced', dle='advanced',
-        full_run=False):
+def setup(dimensions=(50, 50, 50), spacing=(15.0, 15.0, 15.0), tn=500.,
```

Comment:
Are you *also* trying to refactor this file so that it looks like as similar as possible to the other examples?